release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -165,6 +165,20 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-13-NUMA-scheduler-ga"]
+==== NUMA-aware scheduling with the NUMA Resources Operator is generally available
+
+With {product-title} {product-version}, NUMA-aware scheduling with the NUMA Resources Operator is generally available. This was a Technology Preview feature in {product-title} 4.10, 4.11, and 4.12. 
+
+The NUMA Resources Operator deploys a NUMA-aware secondary scheduler that makes scheduling decisions for workloads based on a complete picture of available NUMA zones in clusters. This ensures that latency-sensitive workloads are processed in a single NUMA zone for maximum efficiency and performance. 
```

Comment:
The last sentence here is passive. Consider revising for active voice to follow the ISG https://www.ibm.com/docs/en/ibm-style?topic=style-basics

Reply:
Also, it isn't really clear what "This" is at the start of the second sentence.